%YAML:1.0

cam0_calib: "/home/bo/MonoLidarMapping/src/mono_lidar_mapping/config/hk_cam00.yaml"
cam1_calib: "/home/bo/MonoLidarMapping/src/mono_lidar_mapping/config/hk_cam01.yaml"
image0_topic: "/camera/image_color"
image_width: 1920
image_height: 1200

min_pnp_loop_num: 10
min_brief_loop_num: 25
min_orb_loop_num: 20
skip_time: 0.5
skip_dis: 0.5
skip_cnt: 0
loop_search_time: 0.3
loop_search_gap: 100
debug_image: 1


